backend/climateconnect_api/models/notification.py

```diff
@@ -31,9 +33,11 @@ class Notification(models.Model):
         (POST_COMMENT, "post_comment"),
         (REPLY_TO_POST_COMMENT, "reply_to_post_comment"),
         (GROUP_MESSAGE, "group_message"),
+        (JOIN_PROJECT_REQUEST,"join_project_request"),
+        (PROJECT_JOIN_REQUEST_APPROVED,"project_join_request_approved"),
         (IDEA_COMMENT, "idea_comment"),
         (REPLY_TO_IDEA_COMMENT, "reply_to_idea_comment"),
-        (PERSON_JOINED_IDEA, "person_joined_idea")
+        (PERSON_JOINED_IDEA, "person_joined_idea"),
     )
```

Comment:
Non-Blocker and Note to engineers: I think its time we move notification_type to its on database table. I think this will be expanded to too many different types so better to have them in a database table then hard coded in the codebase.